### PR TITLE
Topology2: Update dummy wov on latest code

### PR DESCRIPTION
**Pull request by libinyang**
Update dummywov on latest code as there are many changes, which make wov not work on the current code.

### CODE REVIEW

File: tools/topology/topology2/platform/intel/dmic-wov.conf [hunk 1]
```diff
@@ -123,6 +123,7 @@ Object.PCM {
 		name	"DMIC16k"
```
Comment (kv2019i) [inline tools/topology/topology2/platform/intel/dmic-wov.conf:123]: typo in commit: topogloy2

**Review by kv2019i — APPROVED** [review]

**Comment by kv2019i** [comment]
Add @ranj063 to check updated tplg2 usage.

File: tools/topology/topology2/include/pipelines/cavs/wov-detect.conf [hunk 2]
```diff
@@ -57,6 +57,7 @@ Class.Pipeline."wov-detect" {
 		# to the host copier. The virtual widget does not exist in
 		# the firmware.
 		virtual."virtual.detect_sink" {
+			name "virtual.detect_sink"
```
Comment (ranj063) [inline tools/topology/topology2/include/pipelines/cavs/wov-detect.conf:60]: virtual. 0 { on line 69 please

File: tools/topology/topology2/platform/intel/test-wov-config.conf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# The sof_abi_hdr for ipc4 is:
```
Comment (ranj063) [inline tools/topology/topology2/platform/intel/test-wov-config.conf:1]: can you please remove this file and add the data block directly into wov.conf like we're doing for the smart amp?

**Comment by libinyang** [comment]
@kv2019i @ranj063 Thanks for the comments. PR has been updated based on the feedback.

File: tools/topology/topology2/include/components/wov.conf [hunk 4]
```diff
@@ -93,8 +93,18 @@ Class.Widget."wov" {
 			put 0
 		}
 		max 4096
-		IncludeByKey.WOV_UUID {
-			'1f:d5:a8:eb:27:78:b5:47:82:ee:de:6e:77:43:af:67'	"platform/intel/test-wov-init-payload.conf"
+		Object.Base.data."1" {
+			IncludeByKey.WOV_UUID {
+				'1f:d5:a8:eb:27:78:b5:47:82:ee:de:6e:77:43:af:67'	{bytes "0x53, 0x4f, 0x46, 0x34, 0x02, 0x00, 0x00, 0x00,
+												0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04,
```
Comment (lgirdwood) [inline tools/topology/topology2/include/components/wov.conf:99]: formatting looks off ?

File: tools/topology/topology2/include/pipelines/cavs/wov-detect.conf (hunk 2 above)
Comment (lgirdwood) [reply tools/topology/topology2/include/pipelines/cavs/wov-detect.conf:60]: @libinyang can you respond to this change ? It looks different to the suggestion from @ranj063 ?

**Review by lgirdwood — COMMENTED** [review]
Just some opens on the widget ID and teh formatting. @libinyang pls respond, should be able to merge tomorrow if all ok.

File: tools/topology/topology2/include/pipelines/cavs/wov-detect.conf (hunk 2 above)
Comment (ranj063) [reply tools/topology/topology2/include/pipelines/cavs/wov-detect.conf:60]: 1 works too @lgirdwood.

**Review by ranj063 — APPROVED** [review]

**Comment by mengdonglin** [comment]
The PR failure in Jenkins CI is a known issue: https://github.com/thesofproject/sof/issues/6734, not caused by this PR.

**Comment by kv2019i** [comment]
Ack @mengdonglin , https://sof-ci.01.org/sofpr/PR7070/build3874/devicetest/index.html?model=ADLP_RVP_NOCODEC_IPC4ZPH&testcase=multiple-pause-resume-5 is the known multiple-pause-resume bug.

Proceeding with merge.

**Pull request merged**